### PR TITLE
Add API key tab to user profile page

### PR DESCRIPTION
**Pull request by bnmajor**
Updated user profile page to be a tabbed view with one tab for user
information and another for managing API keys.

Signed-off-by: Brianna Major <brianna.major@kitware.com>

### CODE REVIEW

**Comment by bnmajor** [comment]
Uses components created in openchemistry/oc-web-components#133

File: src/components/user/user-profile.js [hunk 1]
```diff
@@ -1,12 +1,43 @@
...
+const StyledTabs = withStyles({
+  indicator: {
+    display: 'flex',
+    justifyContent: 'center',
+    backgroundColor: 'transparent',
+    '& > div': {
+      width: '100%',
+      backgroundColor: '#ffffff',
+    },
+  },
```
Comment (cjh1) [inline src/components/user/user-profile.js:19]: Great to us use `withStyles(...)`

**Review by cjh1 — COMMENTED** [review]
@bnmajor This is looking really good! I think we should make the "Permission scope" part configurable. Its great that we have all the options. However, for openchemistry the user isn't necessary going to he familiar with Girder, so it may look a little scary. I think it should be basically reduce to two options "read" and "write". Lets chat about this in person.

File: src/components/user/user-profile.js (hunk 1 above)
Comment (cryos) [reply src/components/user/user-profile.js:19]: Totally agree with keeping this very simple, our target audience will likely have no idea what Girder is, and we should use a simple set of options where reasonable. This is in the wrong place, meant to reply to the read and write level of access...

**Comment by bnmajor** [comment]
@cjh1 Let me know if this looks like what you had in mind. Currently you can choose to pass a list of permissions to the component to select which are available. Passing no list will default to all Girder permissions.

**Review by cjh1 — APPROVED** [review]

**Pull request merged**